docs/start_gradient.md

```diff
@@ -6,7 +6,7 @@ This is a quick guide to getting started with Deep Learning for Coders on Papers
 
 ## Pricing
 
-**Paperspace has a [Free Tier of free GPU and CPU instances](https://docs.paperspace.com/gradient/instances/free-instances).** To use them, choose *Free-GPU* or *Free-P5000* (recommended) in step 2.6 Create Notebook.
+**Paperspace has a [Free Tier of free GPU and CPU instances](https://docs.paperspace.com/gradient/instances/free-instances).** To use them, choose *Free-GPU* or *Free-P5000* (recommended) in step 3.3 Create Notebook.

```

Comment:
*Free-P5000* isn't actually showing up as "free" for me. it seems to require the pro plan ($8/mo). Would you like me to change this as well?